docs/views/templates/check-your-answers.html

```diff
@@ -9,159 +9,178 @@
-          <dt class="app-check-your-answers__question">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
             Home address
           </dt>
-          <dd class="app-check-your-answers__answer">
-            72 Guild Street<br />
-            London<br />
-            SE23 6FH
-          </dd>
-          <dd class="app-check-your-answers__change">
-            <a href="#">
-              Change<span class="govuk-visually-hidden"> home address</span>
-            </a>
+          <dd class="govuk-summary-list__value">
+            72 Guild Street
+            <br>London
+            <br>SE23 6FH
+          </dd>
+          <dd class="govuk-summary-list__actions">
+            <ul class="govuk-summary-list__actions-list">
+              <li class="govuk-summary-list__actions-list-item">
+                <a href="#">
+                  Change
+                  <span class="govuk-visually-hidden"> home address</span>
+                </a>
+              </li>
+            </ul>
           </dd>
         </div>
-
-        <div class="app-check-your-answers__contents">
-          <dt class="app-check-your-answers__question">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
             Licence period
           </dt>
-          <dd class="app-check-your-answers__answer">
-            This is a longer paragraph of text provided by the user to provide additional information.<br /><br />
-            This is a second paragraph of text provided by the user.
-          </dd>
-          <dd class="app-check-your-answers__change">
-            <a href="#">
-              Change<span class="govuk-visually-hidden"> licence period</span>
-            </a>
+          <dd class="govuk-summary-list__value">
+            <p>This is a longer paragraph of text provided by the user to provide additional information.</p>
+            <p>This is a second paragraph of text provided by the user.</p>
+          </dd>
+          <dd class="govuk-summary-list__actions">
+            <ul class="govuk-summary-list__actions-list">
+              <li class="govuk-summary-list__actions-list-item">
+                <a href="#">
+                  Change
+                  <span class="govuk-visually-hidden"> licence period</span>
+                </a>
+              </li>
+            </ul>
           </dd>
         </div>
       </dl>
 
-
       <h2 class="govuk-heading-m">Now send your application</h2>
 
-      <p>
-        By submitting this notification you are confirming that, to the best of your knowledge, the details you are providing are correct.
-      </p>
+      <p>By submitting this notification you are confirming that, to the best of your knowledge, the details you are providing are correct.</p>
 
-      <a class="govuk-button" href="confirmation-page" role="button">
+      <button type="submit" class="govuk-button">
```

Comment:
Not sure what to do here.

I think this should either

1) Be a button inside a form (currently there's no form tag), or

2) Revert to being a link for the purposes of prototyping.

Option 1 is more correct. However it really needs to submit some sort of data to be fully correct. This could be a hidden input:

```
<input type="hidden" name="answers-checked" value="true">
```

Reply:
Sorry this is a mistake in the Design System version that I copied across. Working too fast! I believe all of the other full page examples have the form tag. Will update.

Reply:
I've updated, let me know if that's what you had in mind...

Reply:
looks good thanks!